commands/artworks.go

```diff
@@ -157,7 +157,7 @@ func parseArtworkArgument(arg string) (int, string, bool) {
 	return 0, "", false
 }
 
-func share(b *bot.Bot) func(ctx *gumi.Ctx) error {
+func share(b *bot.Bot, s) func(ctx *gumi.Ctx) error {
```

Comment:
```suggestion
func share(b *bot.Bot, s post.SkipMode) func(ctx *gumi.Ctx) error {
```
Go is a strongly typed language, you need to specify the type of your arguments.